modules/light-clients/07-tendermint/types/update.go

```diff
@@ -259,5 +243,38 @@ func update(ctx sdk.Context, clientStore sdk.KVStore, clientState *ClientState,
 	// set metadata for this consensus state
 	setConsensusMetadata(ctx, clientStore, header.GetHeight())
 
-	return clientState, consensusState
+	return &cs, consensusState, nil
+}
+
+// pruneOldestConsensusState will retrieve the earliest consensus state for this clientID and check if it is expired. If it is,
+// that consensus state will be pruned from store along with all associated metadata. This will prevent the client store from
+// becoming bloated with expired consensus states that can no longer be used for updates and packet verification.
+func (cs ClientState) pruneOldestConsensusState(ctx sdk.Context, cdc codec.BinaryCodec, clientStore sdk.KVStore) {
```

Comment:
nit: add some logical spacing between code in this function.

Reply:
sounds good. Copied from existing code, but agree it could use spacing

Reply:
updated!